app/views/comments/edit.html.erb

```diff
@@ -0,0 +1,19 @@
+<%- if @comment.errors.present? -%>
+  <%- @comment.errors.full_messages.each do |error| -%>
+    <ul>
+      <li><%= error %></li>
+    </ul>
+  <% end %>
+<% end %>
```

Comment:
if you use `<%- something -%>`, then be consistent across the board. your `<% end %>`s don't have dashes, for example.